daonode/src/main/java/bisq/daoNode/endpoints/ProofOfBurnApi.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.daoNode.endpoints;
+
+import bisq.core.dao.state.model.blockchain.Tx;
+
+import bisq.common.util.Hex;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import lombok.extern.slf4j.Slf4j;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+
+
+import bisq.daoNode.DaoNode;
+import bisq.daoNode.DaoNodeRestApiApplication;
+import bisq.daoNode.dto.ProofOfBurnDto;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.Application;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.MediaType;
+
+@Slf4j
+@Path("/proof-of-burn")
+@Produces(MediaType.APPLICATION_JSON)
+@Tag(name = "Proof of burn API")
+public class ProofOfBurnApi {
```

Comment:
I am looking at the [JAX-RS spec](http://download.oracle.com/otn-pub/jcp/jaxrs-1.0-fr-eval-oth-JSpec/jaxrs-1.0-final-spec.pdf).   On Page 17, I see _A resource class is a Java class that uses JAX-RS annotations to implement a corresponding Web resource. Resource classes are POJOs that have at least one method annotated with @Path or a request method designator._

Instead of being called `ProofOfBurnApi`, should it just be `ProofOfBurn` or `ProofOfBurnResource`?
Should the package be renamed too, say `bisq.daoNode.resource`?   @salkiam probably knows.

Reply:
I see the `@io.swagger.v3.oas.annotations.responses.ApiResponse`, so @salokiam could be thinking about swagger in the choice of classname.

Reply:
I discussed naming with @Salokiam and we both agree that resource is kind of overloaded term. Spring use Controller (also overloaded but better IMO). So we though postfixing it with Api is better. But not super stong opinionated here as well.